filter/sanitize.js

```diff
@@ -24,11 +24,21 @@ module.exports = (fields, locations) => {
       middleware[methodName] = (...options) => {
         sanitizers.push({
           sanitizer: sanitizerFn,
+          custom: false,
           options
         });
         return middleware;
       };
     });
 
+  middleware.customSanitizer = (sanitizer, ...options) => {
```

Comment:
To keep the same interface as the `.custom()` validator method, this should not receive any options.